src/main/java/net/pms/util/BitRateMode.java

```diff
@@ -0,0 +1,52 @@
+package net.pms.util;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+import java.util.Locale;
+
+/**
+ * This {@code enum} represents the possible bit rate "modes".
+ */
+public enum BitRateMode {
+
+	/** Constant bit rate mode*/
+	CBR,
+
+	/** Variable bit rate mode */
+	VBR;
+
+	@Override
+	public String toString() {
+		switch (this) {
+			case CBR:
+				return "Constant";
```

Comment:
No need, i've big glasses and i only misread `1` and `l `:stuck_out_tongue: 

An old 14" cathodic screen, but i have colors LOL